tests/core/test_parametric_geometry.py

```diff
@@ -128,6 +129,49 @@ def test_ParametricPseudosphere():
 def test_ParametricRandomHills():
     geom = pv.ParametricRandomHills()
     assert geom.n_points
+    with pytest.warns(PyVistaDeprecationWarning):
+        geom = pv.ParametricRandomHills(numberofhills=30)
+        if pv._version.version_info >= (0, 46):
+            raise RuntimeError('Remove this deprecated parameter')
```

Comment:
While these are nice to prevent the deprecation from sticking around too long.  Didn't it create a release problem last time?  Not sure if it is worth fixing here, or kicking down the road.

Reply:
As you say, there will raise an error in the release and that is not pleasant. However, we are still unable to come up with a better solution.